deep_qa/training/optimizers.py

```diff
@@ -14,18 +14,27 @@
 \* I should also note that Keras is an incredibly useful library that does a lot
 of things really well. It just has a few quirks...
 """
+import logging
 from typing import Union
-from keras.optimizers import SGD, RMSprop, Adagrad, Adadelta, Adam, Adamax, Nadam
+
+# pylint: disable=no-name-in-module
```

Comment:
Hmm, ok.  Tensorflow does its own crazy stuff, so it's not too surprising that it didn't work.